ivy_tests/test_ivy/test_frontends/test_jax/test_random.py

```diff
@@ -1417,3 +1419,99 @@ def call():
     for u, v in zip(ret_np, ret_from_np):
         assert u.dtype == v.dtype
         assert u.shape == v.shape
+
+
+
+# Define the helper function to sample mean and covariance matrix
+def get_mean_cov_vector(shape, dtype):
+    mean_dim_size = shape[-1]
+
+    @st.composite
+    def mean_strategy(draw):
+        mean_size = draw(st.integers(min_value=1, max_value=mean_dim_size))
+        mean_values = draw(st.lists(st.floats(min_value=-10, max_value=10), min_size=mean_size, max_size=mean_size))
+        return np.array(mean_values, dtype=dtype)
+
+    mean = mean_strategy()
+
+    return mean
+
+# Sample the mean
+mean = get_mean_cov_vector(shape=(4,), dtype=np.float64)
+
+@handle_frontend_test(
+    fn_tree="jax.random.multivariate_normal",
+    dtype_key=helpers.dtype_and_values(
+        available_dtypes=["uint32"],
+        min_value=0,
+        max_value=2000,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+        max_dim_size=2,
+    ),
+    shape=helpers.get_shape(
+        min_num_dims=1, max_num_dims=6, min_dim_size=1, max_dim_size=6
+    ),
+    dtype=helpers.get_dtypes("float", full=False),
+    mean=st.lists(st.floats(min_value=-10, max_value=10), min_size=1, max_size=10).map(np.array),
+    cov=helpers.dtype_and_values(
+        available_dtypes=("float32","float64"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x])),
+    ).filter(
+        lambda x: "float16" not in x[0]
+        and "bfloat16" not in x[0]
+        and np.linalg.cond(x[1][0]) < 1 / sys.float_info.epsilon
+        and np.linalg.det(x[1][0]) != 0
+    ),
+    method=st.sampled_from(["cholesky", "eigh", "svd"]),
+)
+def test_jax_multivariate_normal(
+    *,
+    dtype_key,
+    mean,
+    cov,
+    shape,
+    dtype,
+    method,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+):
+    input_dtype, key= dtype_key
+    x = cov
+    x = np.asarray(x[0], dtype=dtype[0])
```

Comment:
`cov` contains both dtype and the matrix, that's why you are getting the error of cannot convert string to float.

Reply:
```
    input_dtype, key= dtype_key
    print("pass the troubles")
    cov_dtype, cov_matrix = cov
    x = np.asarray(cov_matrix[0], dtype=cov_dtype[0])
    # positive definite
    x = np.matmul(x.T, x) + np.identity(x.shape[0]) * 1e-3
```
Now I am extracting the cov matrix and its corresponding dtypes, but I followed the same logic from the cholesky test function in making the matrix positive definite. However, I still get an error saying matrix is not positive definite and will the postive definite for each matrix differ depending on the method?

Reply:
Maybe the non-dtype filters present in the cholesky method can help, did you try those out?